apps/modernization-ui/src/apps/search/terms/removeTerm.ts

```diff
@@ -26,6 +26,8 @@ const removeTerm =
             const adjusted = removeAndTrim(value as string, term.value);
             form.setValue(key, adjusted as PathValue<C, Path<C>>);
         } else {
+            // set the value to null in case there is no form element for it
+            form.setValue(key, null as PathValue<C, Path<C>>);
```

Comment:
This may be the only way to handle clearing the chips for the Event ID's until they are added to the form itself, bummer.